fitbit-react/src/Components/Navbar/Navbar.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import { GrMenu } from "react-icons/gr";
+import "./navbar.css";
+import SearchBar from "./SearchBar";
+import PrimaryButton from "../Buttons/PrimaryButton";
+import Logo from "./Logo";
+import NavbarIcons from "./NavbarIcons";
+
+function Navbar() {
```

Comment:
can change is to ES6 syntax ==>  const Navbar = () => {}

Reply:
Ya i can, but i like my components to be in this manner